README.md

```diff
@@ -31,6 +31,8 @@ Global:
 
  * _System/Status_
    * `fortigate_version_info`
+ * _System/Time_
```

Comment:
I think we can keep this under Status to avoid having a bunch of categories with only one metric in them. We can move it to System/Status/Time later if we want to.

Reply:
For now the probe category name inherit the API URL path, making `System/Time`, `System\Status` or `System\Status\Time` break this naming convention.
Gathering mulitple metrics in a category name can be sometimes easier for inclusion/exclusion but remove the maximum granularity for inclusion/exclusion that we have right now.
Kipping a link between API URL path and category name maximize optimization when including/excluding probes as a API call is done or not depending on a probe being included/excluded.
Therefore no useless API call are done.

That being said, even if I believe in KISS, I am leaning for `System/Status/Time` which keep the link between API call and metric name. Even if this is not exactly the same path, at least there is a dedicated category name.
This has the advantage of permitting prefix matching.
Seems acceptable as we already have small arrangements for example with `BGP/Neighbors/IPv4` 😏
It does not remove a new category name tho. 😏

Do we go for `System/Status/Time` then ?

Reply:
We can go with either `System/Time` or `System/Status/Time` - you make a good point with the API naming. I will leave the choice to you. We might want to expose things like NTP status later so we might add more metrics under the category either way.

Reply:
FYI, in FortiOS v7 NTP has URL path `system/ntp/status`

I think we should tweek probe category name following those rules:
- By default, probe category name should be derived from the API URL path (ex: `api/v2/monitor/user/fsso` become `User/Fsso`)
- A single probe category name should never be a subpart of another probe category name, in order to keep inclusion/exclusion declaration simple.
- probe category name should match : `^([a-ZA-Z0-9]+/)+[a-ZA-Z0-9]+$`
- Removing a prefix part is possible if not conflicting or bringing ambiguity with current and futur probe category name/URL path (ex: `BGP/Neighbo...` instead of `Router/BGP/Neighbo...` -- shorter is better
- Adding a suffix part to avoid prefix matching unwilled inclusion/exclusion is possible (ex: `api/v2/monitor/vpn/ssl` and `api/v2/monitor/vpn/ssl/stats` goes to `VPN/Ssl/Connections` and `VPN/Ssl/Stats`)
- Creating an additional category section not present in the API URL path is possible to make grouping more efficient and anticipate present or futur Fortigate API prefix match issues (ex: `monitor/system/time` conflict with `monitor/system/timezone`, then category name can be `System/Time/Clock` and `System/Time/Timezone`)

I agree that grouping in categories is a good thing. For now probe category name look well sized to me and do respect above rules.
About grouping all time related probes in a category like `System/Time/...` (ex: `System/Time/Clock` and `System/Time/NTP/Status`) actually looks like the best solution. `system/Timezone` made up my mind 😄 (even if `timezone` has very few interest, it shows the kind of problem that could come to us with only URL path rule). 
I think, on the long term, the above rules make things simplier even if we do not 100% follow the Fortigate API URL path we are still pretty close to.  

Will propose a draft PR for Contributing.md to cast those rules in stone 🎵 (#64)

Finally going for `System/Time/Clock`

WDYT @bluecmd ?

Reply:
Sounds good to me :+1: @secustor any issues with the above?

Reply:
@bluecmd  I have nothing to add to the discussion :+1: